src/urllib3/util/retry.py

```diff
@@ -260,12 +271,12 @@ def get_backoff_time(self):
             return 0
 
         backoff_value = self.backoff_factor * (2 ** (consecutive_errors_len - 1))
-        return min(self.BACKOFF_MAX, backoff_value)
+        return float(min(self.BACKOFF_MAX, backoff_value))
```

Comment:
`Returning Any from function declared to return "float"`

Isn't it because the `min` can be called with an iterable. so the return result can be based on the iterable value.
for example:
```
>>> min("abcDEF") # find smallest item in the string
'D'
>>> min([2, -1, 4, 3]) # find smallest item in the list
-1
>>> min(("one", "two", "three")) # find smallest item in the tuple
'one'
>>> min({1: "one", 2: "two", 3: "three"}) # find smallest item in the dict
1
```